### PR TITLE
Fix html files to reflect new repository name

### PR DESCRIPTION
**Pull request by pinealservo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,55 +15,56 @@
...
         </ul>
       </header>
       <section>
 
 <h1>
-<a name="open-avb" class="anchor" href="#open-avb"><span class="octicon octicon-link"></span></a>Open AVB</h1>
+<a name="open-avnu" class="anchor" href="#open-avnu"><span class="octicon octicon-link"></span></a>OpenAvnu</h1>
 
-<p>The Open AVB project is sponsored by the AVnu Alliance.</p>
+<p>The OpenAvnu project is sponsored by the AVnu Alliance.</p>
```
Comment (bdthomsen) [inline index.html]: Please change to "Avnu Alliance"

File: index.html [hunk 2]
```diff
@@ -15,55 +15,56 @@
...
+illustrate the flow - but a professional AVB/TSN system requires a full media stack </p>
 
 <ul>
 <li>including audio and video inputs and outputs, media processing elements, and 
 various graphical user interfaces. Various companies provide such integrated 
 solutions.</li>
 </ul>
 
-<p>For more information about AVB, see also the AVnu Alliance webpage at
+<p>For more information about AVB/TSN, see also the AVnu Alliance webpage at
```
Comment (bdthomsen) [inline index.html]: Please change to "Avnu Alliance"

**Comment by pinealservo** [comment]
Made the changes @bdthomsen suggested

**Comment by andrew-elder** [comment]
Thanks!

**Pull request merged**